src/battle/overlay_12_0224E4FC.c

```diff
@@ -1352,17 +1353,17 @@ BOOL ov12_02250490(BattleSystem *bsys, BATTLECONTEXT *ctx, int *out) {
     return ret;
 }
 
-int ov12_022506D4(BattleSystem *bsys, BATTLECONTEXT *ctx, int battlerIdAttacker, u16 moveNo, int a4, int a5) {
-    int battlerIdTarget = 0xFF;
-    int unkA;
+int ov12_022506D4(BattleSystem *bsys, BATTLECONTEXT *ctx, int battlerIdAttacker, u16 moveNo, int a4, int range) {
+    int battlerIdTarget = BATTLER_NONE;
+    int moveRange;
     
     if (moveNo) {
-        unkA = ctx->unk_334.moveData[moveNo].unk8;
+        moveRange = ctx->unk_334.moveData[moveNo].range;
     } else {
-        unkA = a5;
+        moveRange = range;
     }
     
-    if (unkA == 4) {
+    if (moveRange == 4) {
```

Comment:
range constants? what does each value mean?

Reply:
Whomst it targets, I should have enough context clues to figure them all out? Maybe not all of them but the ones I use